mobile/lib/features/welcome/payout_chart.dart

```diff
@@ -0,0 +1,292 @@
+                        return DropdownMenuItem<int>(
+                          value: value,
+                          child: Text("\$$value"),
+                        );
+                      }).toList(),
+                    )
+                  ],
+                ),
+                Row(
+                  children: [
+                    const Text('Leverage'),
+                    const SizedBox(
+                      width: 5,
+                    ),
+                    DropdownButton<int>(
+                      value: leverageDropDownChoice,
+                      elevation: 16,
+                      style: const TextStyle(color: Colors.deepPurple),
+                      underline: Container(
+                        height: 2,
+                        color: Colors.deepPurpleAccent,
+                      ),
+                      onChanged: (int? value) {
+                        setState(() {
+                          leverageDropDownChoice = value!;
+                        });
+                      },
+                      items: leverageChoices.map<DropdownMenuItem<int>>((int value) {
+                        return DropdownMenuItem<int>(
+                          value: value,
+                          child: Text(
+                            "${value}x",
+                            textAlign: TextAlign.end,
+                          ),
+                        );
+                      }).toList(),
+                    )
+                  ],
+                ),
+              ],
+            ),
+          ],
+        ),
+        SizedBox(
+          height: 230,
+          child: Center(
+            child: AspectRatio(
+              aspectRatio: 1.9,
+              child: Padding(
+                padding: const EdgeInsets.only(left: 20, right: 40),
+                child: LayoutBuilder(
+                  builder: (context, constraints) {
+                    return LineChart(
+                      LineChartData(
+                        minX: 0,
+                        maxX: maxPrice,
+                        lineTouchData: LineTouchData(
+                          touchTooltipData: LineTouchTooltipData(
+                            maxContentWidth: 100,
+                            tooltipBgColor: Colors.transparent,
+                            getTooltipItems: (touchedSpots) {
+                              return touchedSpots.map((LineBarSpot touchedSpot) {
+                                final textStyle = TextStyle(
+                                  color:
+                                      touchedSpot.bar.gradient?.colors[0] ?? touchedSpot.bar.color,
+                                  fontWeight: FontWeight.bold,
+                                  fontSize: 14,
+                                );
+                                return LineTooltipItem(
+                                  'at ${formatter.format(touchedSpot.x)} \n you get ₿${touchedSpot.y.toStringAsFixed(8)}',
```

Comment:
🔧 This tooltip doesn't look so good in the video. It always overlaps with some other element of the UI.

Reply:
I leave these changes for now and let this be reworked when we use this component in a different setting.